engine/src/main/java/org/camunda/bpm/engine/ManagementService.java

```diff
@@ -1343,4 +1343,7 @@ Batch setJobRetriesAsync (List<String> processInstanceIds,
    */
   SchemaLogQuery createSchemaLogQuery();
 
+  /** Enable/disable telemetry */
+  void configureTelemetry(boolean enabled);
```

Comment:
```suggestion
  void enableTelemetry(boolean enabled);
```
The term `configure` makes from my point of view only sense if there is more to specify than just enable or disable the feature. If we used a fluent builder pattern instead, I would go for `configure` otherwise `enable`.